code/game/objects/items/crayons.dm

```diff
@@ -99,7 +99,93 @@
 		if(uses)
 			uses -= 5
 			if(uses <= 0)
-				user << "<span class='warning'>You ate your crayon!</span>"
+				to_chat(user,"<span class='warning'>You ate your crayon!</span>")
 				qdel(src)
 	else
 		..()
+
+/obj/item/weapon/pen/crayon/marker/black
+	icon_state = "markerblack"
+	colour = "#2D2D2D"
+	shadeColour = "#000000"
+	colourName = "black"
+
+/obj/item/weapon/pen/crayon/marker/red
+	icon_state = "markerred"
+	colour = "#DA0000"
+	shadeColour = "#810C0C"
+	colourName = "red"
+
+/obj/item/weapon/pen/crayon/marker/orange
+	icon_state = "markerorange"
+	colour = "#FF9300"
+	shadeColour = "#A55403"
+	colourName = "orange"
+
+/obj/item/weapon/pen/crayon/marker/yellow
+	icon_state = "markeryellow"
+	colour = "#FFF200"
+	shadeColour = "#886422"
+	colourName = "yellow"
+
+/obj/item/weapon/pen/crayon/marker/green
+	icon_state = "markergreen"
+	colour = "#A8E61D"
+	shadeColour = "#61840F"
+	colourName = "green"
+
+/obj/item/weapon/pen/crayon/marker/blue
+	icon_state = "markerblue"
+	colour = "#00B7EF"
+	shadeColour = "#0082A8"
+	colourName = "blue"
+
+/obj/item/weapon/pen/crayon/marker/purple
+	icon_state = "markerpurple"
+	colour = "#DA00FF"
+	shadeColour = "#810CFF"
+	colourName = "purple"
+
+/obj/item/weapon/pen/crayon/marker/mime
+	icon_state = "markermime"
+	desc = "A very sad-looking marker."
+	colour = "#FFFFFF"
+	shadeColour = "#000000"
+	colourName = "mime"
+	uses = 0
+
+/obj/item/weapon/pen/crayon/marker/mime/attack_self(mob/living/user as mob) //inversion
+	if(colour != "#FFFFFF" && shadeColour != "#000000")
+		colour = "#FFFFFF"
+		shadeColour = "#000000"
+		to_chat(usr,"You will now draw in white and black with this marker.")
+	else
+		colour = "#000000"
+		shadeColour = "#FFFFFF"
+		to_chat(usr,"You will now draw in black and white with this marker.")
+	return
+
+/obj/item/weapon/pen/crayon/marker/rainbow
+	icon_state = "markerrainbow"
+	colour = "#FFF000"
+	shadeColour = "#000FFF"
+	colourName = "rainbow"
+	uses = 0
+
+/obj/item/weapon/pen/crayon/marker/rainbow/attack_self(mob/living/user as mob)
+	colour = input(user, "Please select the main colour.", "Marker colour") as color
+	shadeColour = input(user, "Please select the shade colour.", "Marker colour") as color
+	return
+
+/obj/item/weapon/pen/crayon/marker/attack(mob/M as mob, mob/user as mob)
+	if(M == user)

```

Comment:
Maybe we could have an intent for this, possibly? I really don't want to end up accidentally clicking on myself and eating a marker.

Reply:
There are very many items you can harm yourself with on accident regardless of intent. Just be careful. As a pro-tip, you don't have to click the tile you're on to draw on it, you can draw on adjacent tiles.

Reply:
Add drawing faces on people in sharpie when?

Reply:
Now that I can get behind.

Reply:
![img](https://vignette.wikia.nocookie.net/p__/images/5/56/Jigglypuff_preparing_to_scribble_on_her_sleeping_audience.png/revision/latest?cb=20150529192911&path-prefix=protagonist)